src/components/RadialSlider/ButtonContent.tsx

```diff
@@ -11,6 +11,8 @@ const ButtonContent = (props: ButtonProps) => {
       activeOpacity={0.7}
       disabled={disabled}
       onPress={onPress}
+      onLongPress={props.onLongPress}
```

Comment:
@parth-simform Please use optional chaining (?.) and also check default values for onLongPress and onPressOut, This shouldn't crash in any case

Reply:
Resolved